config/jest/browser/close.mjs

```diff
@@ -1,13 +1,8 @@
 import { teardown } from 'jest-environment-puppeteer'
 
-import serverStop from '../server/stop.mjs'
-
 /**
  * Close browser
  */
-export default async function browserClose () {
-  await Promise.all([
-    serverStop(), // Stop web server
-    teardown() // Close browser
-  ])
+export default async function browserOpen () {
```

Comment:
**issue** I'm not sure why the named changed there, that function still closes the browser 😄 

```suggestion
export default async function browserClose () {
```

**suggestion** I don't think there's any need to `await` inside the function, we can just have it be a plain (as in non-`async`) function that returns the `Promise` returned by `teardown` 😄 Or even just export `teardown` as the default export.

Reply:
Oh nice spot on that copy-paste-typo!

Reply:
Will mark this as resolved and reply about Promises in https://github.com/alphagov/govuk-frontend/pull/3300#discussion_r1111799019